targets/targets.json

```diff
@@ -1917,6 +1917,7 @@
             "SKIP_SYSCLK_INIT",
             "FSL_FEATURE_PHYKSZ8081_USE_RMII50M_MODE",
             "SDRAM_IS_SHAREABLE",
+            "FSL_SDK_ENABLE_DRIVER_CACHE_CONTROL=1",
```

Comment:
Not sure about turning this on.  This activates code in fsl_enet.c which mostly isn't used, but I can see one incorrect one at line 812:

           DCACHE_InvalidateByRange((uint32_t)rxBuffer,  (buffCfg->rxBdNumber * rxBuffSizeAlign));

That's assuming `rxBuffer` is a contiguous array of packets (as its pointer type suggests), when really it's pointing to an array of pointers to packets, and accessed via insane casting:

           for (count = 0; count < buffCfg->rxBdNumber; count++)
            {
                /* Set data buffer and the length. */
                curBuffDescrip->buffer = (uint8_t *)(*((uint32_t *)(rxBuffer + count * 4)));

So that invalidate could be trashing quite a lot of system workspace.

You actually want 

           DCACHE_InvalidateByRange((uint32_t)curBuffDescrip->buffer,  rxBuffSizeAlign);

inside the loop.

Reply:
Thanks for highlighting this, I will raise a ticket against the SDK driver.